src/main.rs

```diff
@@ -111,13 +111,14 @@ fn cli() -> Command<'static> {
             DisplayOrder::GetStateRootHash as usize,
         ))
         .subcommand(GetEraInfo::build(DisplayOrder::GetEraInfo as usize))
-        .subcommand(
-            QueryGlobalState::build(DisplayOrder::QueryGlobalState as usize).alias("query-state"),
-        )
+        .subcommand(QueryGlobalState::build(
+            DisplayOrder::QueryGlobalState as usize,
+        ))
+        .subcommand(QueryBalance::build(DisplayOrder::QueryBalance as usize))
         .subcommand(GetDictionaryItem::build(
             DisplayOrder::GetDictionaryItem as usize,
         ))
-        .subcommand(GetBalance::build(DisplayOrder::GetBalance as usize))
+        // .subcommand(GetBalance::build(DisplayOrder::GetBalance as usize))
```

Comment:
Some commented out code that can be removed

Reply:
Good catch!  Fixed in [4b4e21e](https://github.com/casper-ecosystem/casper-client-rs/pull/27/commits/4b4e21ee9851c8616be602d7d4c96a92a5956d8c).